src/Xamarin.Android.Build.Tasks/Tasks/Aapt.cs

```diff
@@ -371,7 +371,7 @@ protected void LogEventsFromTextOutput (string singleLine, MessageImportance mes
 					line = int.Parse (match.Groups["line"].Value) + 1;
 				var level = match.Groups["level"].Value;
 				var message = match.Groups ["message"].Value;
-				if (level.Contains ("warning")) {
+				if (singleLine.Contains ("fakeLogOpen") || level.Contains ("warning")) {
```

Comment:
I think instead of `singleLine.Contains ("fakeLogOpen")` we should use `message.Contains ("fakeLogOpen")`, so that if we wind up with e.g. a filename of `fakeLogOpen` we don't inadvertently grab it and mishandle things.